transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -289,26 +320,10 @@ static jobject _recvFrom(JNIEnv* env, jint fd, void* buffer, jint pos, jint limi
     return createDatagramSocketAddress(env, &addr, res);
 }
 
-static void netty_unix_socket_optionHandleError(JNIEnv* env, int err, char* method) {
-    if (err == EBADF) {
```

Comment:
Please revert these changes as it seems this just moves code and so is not related